src/short-urls/helpers/ShortUrlsRowMenu.tsx

```diff
@@ -45,7 +45,7 @@ const ShortUrlsRowMenu: FCWithDeps<ShortUrlsRowMenuConnectProps, ShortUrlsRowMen
   const [isDeleteModalOpen,, openDeleteModal, closeDeleteModal] = useToggle();
   const visitsComparison = useVisitsComparisonContext();
   const redirectRulesAreSupported = useFeature('shortUrlRedirectRules');
-  const { confirmDeletions } = useSetting('shortUrlsList', { confirmDeletions: true });
```

Comment:
This worked as expected as long as `shortUrlsList` was not set at all, but as soon as it is set, the fallback is skipped, and we end up with an object where `confirmDeletions` is `undefined`, which is then treated as falsy.

This ensures `confirmDeletions` has a default value of `true`, even if the `shortUrlsList` object was already set.